docs/guide/config-composer-json.md

```diff
@@ -0,0 +1,135 @@
+## Setting up your composer.json <span id="config-composer-json"></span>
+
+There are several ways to use the assets in Yii, you can use it in the traditional way in Yii2 by placing the Bower and NPM dependencies, in this case all packages are downloaded from:
+
+- [AssetPackagist](https://asset-packagist.org/).
+
+Directory structure:
+
+- vendor
+  - bower
+  - npm
+
+```    
+{
+    "name": "assets/assets-bootbox",
+    "type": "library",
+    "minimum-stability": "dev",
+    "require": {
+        "php": "^7.2.0",
+        "npm-asset/bootbox": "@dev"
+    },
+    "config": {
+        "process-timeout": 1800,
+    },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
+        }
+    ]
+}
+```
+
+Another option is to move the assets of the vendor directory to the /node_modules directory, we can do it in two ways, the first one maintaining the dependence of <strong>AssetPackagist</strong> and the second way using <strong>foxy with hidev-composer-plugin</strong>, this will allow the assets to be downloaded from <strong>npm</strong>.
```

Comment:
```suggestion
Another option is to move the assets of the vendor directory to the /node_modules directory, we can do it in two ways, the first one maintaining the dependence of **AssetPackagist** and the second way using **foxy with hidev-composer-plugin**, this will allow the assets to be downloaded from **npm**.
```